apostrophecms/i18n-static
# Pro 3235

## Summary

i18n-static plugin for A3

## What are the specific steps to test this change?

1. `npm link @apostrophecms/i18n-static` in testbed
2. add `"@apostrophecms/i18n-static": "1.0.0"` in the dependencies in testbed package.json
3. add 
```
'@apostrophecms/i18n-static': {
      options: {
        excludeNamespaces: [ 'aposEvent', 'aposForm' ]
      }
    },
``` 
in testbed app.js (without `excludeNamespaces` option if you want all namespaces in the editor modal)
4. Start the project, a new entry in the admin bar is displayed: i18n static phrases
5. Create some, export them, import them..

## What kind of change does this PR introduce?
*(Check at least one)*


## Make sure the PR fulfills these requirements:


If adding a new feature without an already open issue, it's best to open a **feature request issue** first and wait for approval before working on it.

**Other information:**
